carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/sampling/SamplingConfiguration.kt

```diff
@@ -23,12 +23,27 @@ interface SamplingConfiguration
  * as part of setting up a [DeviceDescriptor].
 */
 @DeviceDescriptorBuilderDsl
-interface SamplingConfigurationBuilder
+interface SamplingConfigurationBuilder<TSamplingConfiguration : SamplingConfiguration>
```

Comment:
I would like generic types to be one capital letter as they are just a placeholder. The very verbose type labels actually make it less legible

Reply:
Mmmm ... agree to disagree. :) If there are generic type constraints, I do prefer to keep something akin to an interface name in code. Especially when there are multiple generics (not the case in this class, though, but there is a `TBuilder` lower in the file).

Reply:
`TConfig` then

Reply:
Resolved in next PR (which targets this PR).